text/0000-github-access-policy.md

```diff
@@ -0,0 +1,58 @@
+- Feature Name: rust-lang_github_org_access_policy 
+- Start Date: 2020-03-02 
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a policy for managing permissions to the [Rust-Lang GitHub Organization](https://www.github.com/rust-lang) and repositories within this organization.
+
+This RFC was written in consultation with the Governance Working Group and the Infrastructure team. Most discussion took place on [this issue](https://github.com/rust-lang/wg-governance) and [this pull request](https://github.com/rust-lang/wg-governance/pull/42).
+
+# Motivation
+[motivation]: #motivation
+
+Access control for the [Rust-Lang GitHub Organization](https://www.github.com/rust-lang) and repositories within that organization is currently managed ad-hoc. We need a policy that defines how these accesses are granted and managed. This will allow us to have greater security in permissions to our GitHub org and also allow the infra team to build appropriate tooling to automate access control when possible.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Rust-Lang GitHub Permissions Policy
+This policy applies to both the [Rust-Lang GitHub Organization](https://github.com/rust-lang/) and all repositories within that organization.
+
+### Rust-Lang Organization
+Membership in the Rust-Lang GitHub organization is managed by the organization owners.
+
+All members of the [Core Team](https://github.com/rust-lang/team/blob/master/teams/core.toml) have the right to be organization owners.
+
+Selected members of the [Infrastructure Team](https://github.com/rust-lang/team/blob/master/teams/infra.toml) can also be organization owners if their work requires it.
+
+Owners should use a separate account from their main GitHub account dedicated to managing the organization. This account may not be used to commit code and must have 2FA enabled.
```

Comment:
This recommendation came from Mozilla's security team, I'm bringing this feedback to them and asking what they recommend :)

Reply:
The key part of the statement is "no more than one free Account". GitHub will happily take your money for multiple "Pro" accounts. 😄 

I'd suggest using your free account for the org-owner account, as no non-administrative work should be done there. Take advantage of the Pro features on your every-day-work account.

Reply:
We have a standing meeting with GitHub (next one is next week) and I will bring this up with them.

Reply:
I just brought this up with GitHub. The purpose of the part of their ToS (limiting one free account to one person) is to prevent abuse and our use case (using it to add an extra layer of security) certainly does not fall under abuse. While we can't get them to change the wording the ToS (nor do we want them to), I don't think it's something they are likely to take action against us to enforce that line of the ToS. I'm inclined to go ahead and proceed with this and, should anything ever change, we can migrate to a different solution.